src/Network/Network/Az.Network.psd1

```diff
@@ -123,7 +123,9 @@ CmdletsToExport = 'Add-AzApplicationGatewayAuthenticationCertificate',
                'New-AzApplicationGatewayFirewallPolicyManagedRuleOverride', 
                'New-AzApplicationGatewayFirewallPolicyManagedRuleGroupOverride', 
                'New-AzApplicationGatewayFirewallPolicyManagedRuleSet', 
-               'New-AzApplicationGatewayFirewallPolicySetting', 
+               'New-AzApplicationGatewayFirewallPolicySetting',
+               'New-AzApplicationGatewayFirewallPolicySettingLogScrubbingConfiguration',
```

Comment:
Again, could we follow the same naming convention as NRP?

Reply:
same as above